app/viewmodels/checkAnswers/ShortageOrExcessItemSummary.scala

```diff
@@ -0,0 +1,122 @@
+import pages.unsatisfactory.individualItems.{ItemShortageOrExcessPage, WrongWithItemPage}
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{HtmlContent, Text}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{ActionItem, SummaryListRow}
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+import views.html.components.link
+
+import javax.inject.Inject
+
+class ShortageOrExcessItemSummary @Inject()(link: link) {
+
+  def rows(idx: Int, item: MovementItem)(implicit request: DataRequest[_], messages: Messages): Seq[SummaryListRow] = {
+    Seq(
+      shortageOrExcessRow(idx),
+      amountOfShortageOrExcessRow(idx, item),
+      shortageOrExcessInformationRow(idx)
+    ).flatten
+  }
+
+  private def shortageOrExcessRow(idx: Int)(implicit request: DataRequest[_], messages: Messages): Option[SummaryListRow] = {
+    if (isShortageOrExcess(idx)) {
+      request.userAnswers.get(ItemShortageOrExcessPage(idx)).map(
+        shortageOrExcess =>
+          SummaryListRowViewModel(
+            key = s"${ItemShortageOrExcessPage(idx)}.checkYourAnswers.shortageOrExcess.label",
+            value = ValueViewModel(Text(shortageOrExcess.wrongWithItem.toString)),
+            actions = Seq(
+              shortageOrExcessChangeAction(idx, "shortageOrExcess")
+            )
+          )
+      )
+    } else {
+      None
+    }
+  }
+
+  private def amountOfShortageOrExcessRow(idx: Int, item: MovementItem)(implicit request: DataRequest[_], messages: Messages): Option[SummaryListRow] = {
+    if (isShortageOrExcess(idx)) {
+      request.userAnswers.get(ItemShortageOrExcessPage(idx)).map(
+        shortageOrExcess =>
+          SummaryListRowViewModel(
+            key = s"${ItemShortageOrExcessPage(idx)}.checkYourAnswers.amount.label",
+            //TODO: Hardcoded to kg, should be determined from the reference data based on the CN Code in future story
+            value = ValueViewModel(
+              messages(
+                s"${ItemShortageOrExcessPage(idx)}.checkYourAnswers.amount.value",
+                shortageOrExcess.amount.toString,
+                messages(s"unitOfMeasure.$Kilograms.long")
+              )
+            ),
+            actions = Seq(
+              shortageOrExcessChangeAction(idx, "amount")
+            )
+          )
+      )
+    } else {
+      None
+    }
+  }
+
+  private def shortageOrExcessInformationRow(idx: Int)(implicit request: DataRequest[_], messages: Messages): Option[SummaryListRow] = {
+    if (isShortageOrExcess(idx)) {
+      request.userAnswers.get(ItemShortageOrExcessPage(idx)).map(
+        shortageOrExcess =>
+          shortageOrExcess.additionalInfo match {
+            case Some(value) if value != "" =>
+              SummaryListRowViewModel(
+                key = s"${ItemShortageOrExcessPage(idx)}.checkYourAnswers.additionalInfo.label",
+                value = ValueViewModel(Text(value)),
```

Comment:
shortage or excess reason value that is displayed on the check answers item page needs to start with uppercase